.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "10"
+before_install:
```

Comment:
we should install all our dependencies in the install step, later on we will run tests here and maybe publish the app as well

Reply:
if it takes too long, then we need to consider some caching

Reply:
fixed